### DISCUSSION TITLE
量算工具测算水平距离会出现垂直线

### DISCUSSION THREAD

**Discussion by Fang-yl** [zouyaoji/vue-cesium#644] [Q&A]
![Uploading 1735032043744.jpg…]()

**Comment by zouyaoji** [comment]
图裂开了。

**Reply by Fang-yl** [reply]
![1735032043744](https://github.com/user-attachments/assets/0033c2ff-4258-4cf9-98b0-7e0c06188cb8)

**Reply by Fang-yl** [reply]
![1735091083881](https://github.com/user-attachments/assets/507dda28-df81-409e-8b52-8a4bcaebffe2)
在 vc-graphics-polygon 也无法绘制

**Comment by zouyaoji** [comment]
弄个在线能复现的看看

**Reply by Fang-yl** [reply]
![1735091492202](https://github.com/user-attachments/assets/34ddffb0-7402-4ea9-84d1-f2d47cf77da9)
目前还在开发环境，代码如上，大佬看看能不能解决。
